### DISCUSSION TITLE
Generate Flashcards

### DISCUSSION THREAD

**Discussion by briansunter** [briansunter/logseq-plugin-gpt3-openai#165] [Ideas]
- [ ] Generate flashcards from text, topic. 
- [ ] Cloze card generation

**Comment by thiswillbeyourgithub** [comment]
Hi, your project got me interested in NLP after a while of not using it.

So you might be interested in a project I was involved with in the past : https://github.com/paulbricman/autocards/

Also, I made my own personnal script to prompt GPT 3 after seeing your script, it includes a dumb prompt to create cloze style answer that fit my own weird anki formatting : https://github.com/thiswillbeyourgithub/prompt_GPT3

In your opinion, to create multiple flaschards on a given paragraph, is it better to use a custom prompt (eating a lot of tokens each time) or to fine tune GPT3? I have not read enough about that to understand if it's worth the cost.

Is it a terrible idea to have a small dataset of a 10 questions generated from a bunch of paragraph, ask GPT3 to generate *more* examples of questions to paragraph, then use this small dataset to finetune itself to have it ready for your task? Just thinking aloud :)
